content/en/contribute/medic/product-development-process/transparency-accountability.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Transparency & Accountability"
+linkTitle: "Transparency & Accountability"
+weight: 7
+description: >
+  How we make it easy for others to know what's going on and getting done.
+---
+
+It is important that people around Medic, our partners, and community can always see what is being worked on and that commitments are made and kept. There are a few key ways teammates are expected to operate to enable such transparency and accountability.
+
+## GitHub project boards
+Each FWG has a board where tickets are displayed and actively kept up-to-date. These boards are publicly readable and anyone should be able to see the items on the board, what state they are in, and have an expectation of when they will be done.
+
+At the start of the week tickets should be in the "In Progress" state, which means we are committed to getting it done and into the "Done" state by the end of the week.
+
+Boards are:
+- [Allies](https://github.com/orgs/medic/projects/134/views/3)
+- [Care Teams](https://github.com/orgs/medic/projects/259/views/11)
+- [Ecosystem](https://github.com/orgs/medic/projects/134/views/11)
+- [Infrastructure](https://github.com/orgs/medic/projects/134/views/26)
```

Comment:
Sorry for jumping in into this PR. Can we also add the Community Team to the list of boards?

Reply:
> Sorry for jumping in into this PR

Definitely always welcome! ...and yes.